.github/workflows/test.yml

```diff
@@ -0,0 +1,15 @@
+on:
+  push:
+    branches:
+      - master
+  pull_request: {}
+name: test
+jobs:
+  test:
+    runs-on: ubuntu-18.04
+    steps:
+    - uses: actions/checkout@v1
+    - uses: actions/setup-node@v1
```

Comment:
why is it setup-node here? Maybe actions/setup-go?

Reply:
Oh, big oops :)